### ISSUE TITLE
[Design] My VA: Review screenreader functionality on My VA

### ISSUE THREAD

**Issue by ajakabcin** [department-of-veterans-affairs/va.gov-team#57395]
## Background
Per feedback from design intent, midpoint review and staging review, we need to review heading hierarchy and styling to make sure we're in alignment with the style and a11y guidance. Specifically we need to review and test screenreader functionality in the Education and training section and in any scenario where we display "You have [...] to show".

## Tasks
- [ ] Conduct testing on **Education and training** section: Double-check in Angela Fowler's office hours or with other screenreader user to ensure that there's no need for a hidden h3 for the secondary links in the Education and training section
- [ ] Conduct testing on "You have no [...] to show" message that appears under section headings to ensure that screenreader-only H3s are not causing confusion or repetition for screenreader users
- [ ] Once testing is complete work with PM to determine what, if any, FE changes need to be made

## Acceptance Criteria
- Testing and/or review with appropriate a11y specialists on the sections described above is complete
- Tickets are created for FE work (if needed)

## Resources
Tickets from collab cycle touch points calling for review of screenreader functionality:
- #48676 (second point re screenreader and H2/H3)
- #54880

**Comment by ajakabcin** [comment]
Closing - duplicate of #57398
